tracer/test/test-applications/integrations/Samples.RuntimeMetrics/Program.cs

```diff
@@ -20,6 +20,15 @@ private static void Main()
 
             new Thread(GenerateEvents) { IsBackground = true }.Start();
 
+            /*
+             * We need runtime metrics to be refreshed at least twice to have the contention metrics
+             * (because it needs a previous value to compute the variation).
+             * Because of the asynchronous initialization of the performance counters, we sometimes
+             * miss the first refresh. 
+             * 
+             * We wait 30 seconds here to ensure that we'll have at least two refreshes.
+             */
+
```

Comment:
Found and copied this message from one of Kevin's PRs that increased this from 20s to 30s

Since this causes such long tests (3x 30s) it should probably be documented as to _why_ as it wasn't obvious to me at first glance (but found easily via the history of the file)